### PR TITLE
[Oy2-9908] Measure 24: PCR-HH

### PR DESCRIPTION
**Pull request by jphechter**
## Purpose

Create the measure PCR-HH following the pattern established in PCR-AD.

#### Linked Issues to Close

[Oy2-9908](https://qmacbis.atlassian.net/browse/OY2-9908?atlOrigin=eyJpIjoiZWY3Y2JkMGQwMDNkNGU4OTg4Y2RlZGQ3NDhjYmM1NjMiLCJwIjoiaiJ9)

## Approach

Reused most of the work done to create PCR-AD. Made a few tweaks to expand specific PCR-AD checks to cover both PCR-XX measures. Moved all PCR-XX validations into globalValidations directory.

## Assorted Notes/Considerations

There are a couple places in the code I called out that might be good candidates for refactor. Many of these are in places where we specifically check `if PCR-AD then do...`. Increased reliance on these checks is still a good candidate for refactor IMO.

Additionally, we need to use measure specific versions of the Performance Measure and OMS sections. As we will soon have other measures that rely on unique NDR-type components, we should consider the ability to pass-in or specify which NDR-type component to use.

#### Pull Request Creator Checklist

- [X] This PR has an associated issue or issues.
- [X] The associated issue(s) are linked above.
- [X] This PR meets all acceptance criteria for those issues.
- [X] This PR and linked issue(s) are adequately documented
- [X] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [X] Someone has been assigned this PR.
- [X] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: services/ui-src/src/measures/2021/PCRAD/data.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import { DataDrivenTypes } from "../../CommonQuestions/types";
```
Comment (jphechter) [inline services/ui-src/src/measures/2021/PCRAD/data.ts:1]: Changes made to PCR-AD were explicitly done for the purposes of cleanup or clarification. Nothing here affects how the measure actually works.

File: services/ui-src/src/components/PCRRate/index.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ interface Props extends QMR.InputWrapperProps {
   customMask?: RegExp;
 }
 
-export const MultiRate = ({
+export const PCRRate = ({
```
Comment (jphechter) [inline services/ui-src/src/components/PCRRate/index.tsx:20]: Renamed for clarity.

File: services/ui-src/src/measures/globalValidations/index.ts [hunk 3]
```diff
@@ -1,3 +1,8 @@
 export * from "./dataDrivenTools";
 export * from "./validationsLib";
 export * from "./omsValidationsLib";
+
+// PCR-XX Specific Validations
+export { PCRatLeastOneRateComplete } from "./PCRValidations/PCRatLeastOneRateComplete";
+export { PCRnoNonZeroNumOrDenom } from "./PCRValidations/PCRnoNonZeroNumOrDenom";
+export { PCRvalidateAtLeastOneNDRInDeviationOfMeasureSpec } from "./PCRValidations/PCRvalidateAtLeastOneNDRInDeviationOfMeasureSpec";
```
Comment (jphechter) [inline services/ui-src/src/measures/globalValidations/index.ts:8]: Moved all validations out of PCR-AD into globalValidations. This structure is consistent with how @daniel-belcher is refactoring validations.

**Review by abelzer212 — DISMISSED** [review]

File: services/ui-src/src/measures/2021/PCRHH/index.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            </>
+          )}
+          {isOtherMeasureSpecSelected && <CMQ.OtherPerformanceMeasure />}
+          <CMQ.CombinedRates healthHomeMeasure />
+          {showOptionalMeasureStrat && (
+            <PCRHHOptionalMeasureStrat
+              performanceMeasureArray={performanceMeasureArray}
+              qualifiers={PMD.qualifiers}
+              categories={PMD.categories}
+              adultMeasure
```
Comment (jdinh8124) [inline services/ui-src/src/measures/2021/PCRHH/index.tsx:64]: Not an adult measure

Comment (jphechter) [reply services/ui-src/src/measures/2021/PCRHH/index.tsx]: included. thank you!

**Review by abelzer212 — APPROVED** [review]

**Pull request merged**